Tests/Rules/AvoidUnInitializedVarsInNewRunspaces.tests.ps1

```diff
@@ -0,0 +1,90 @@
+$directory = Split-Path -Parent $MyInvocation.MyCommand.Path
+$testRootDirectory = Split-Path -Parent $directory
+
+Import-Module (Join-Path $testRootDirectory "PSScriptAnalyzerTestHelper.psm1")
+
+$ruleName = "PSAvoidUnInitializedVarsInNewRunspaces"
+
+$settings = @{
+    IncludeRules = @($ruleName)

```

Comment:
I know some old tests do this but since `ruleName` is not re-used a 2nd time, I would not declare a variable for it and just inline it

Reply:
Apparently I peeked at an old test for template :-) will inline it.